tensorrt_llm/llmapi/serialization.py

```
@@ -0,0 +1,224 @@
+    "tensorrt_llm.serve.postprocess_handlers": [
+        "chat_response_post_processor", "chat_stream_post_processor",
+        "completion_stream_post_processor",
+        "completion_response_post_processor", "CompletionPostprocArgs",
+        "ChatPostprocArgs"
+    ],
+    "torch._utils": ["_rebuild_tensor_v2"],
+    "torch.storage": ["_load_from_bytes"],
+}
+
+
+def _register_class(dict, obj):
```

Ruff (0.16.0)

[error] Function argument `dict` is shadowing a Python builtin

(A002)